tests/ballerina-unit-test/src/test/resources/test-src/services/dispatching/uri-template-default.bal

```diff
@@ -53,3 +54,9 @@ service serviceWithNoAnnotation on testEP {
         _ = caller->respond(res);
     }
 }
+
+service on mockEP {
```

Comment:
Shall we add a test case with ServiceConfig(basePath not required) as well?
Because flow is different when ServiceConfig is available.